core/renderers/common/renderer.js

```diff
@@ -139,6 +148,64 @@ Blockly.blockRendering.Renderer.prototype.getConstants = function() {
     (this.constants_));
 };
 
+/**
+ * Get any renderer specific CSS to inject when the renderer is initialized.
+ * @return {!Array.<string>} Array of CSS strings.
+ * @protected
+ */
+Blockly.blockRendering.Renderer.prototype.getCSS_ = function() {
+  var selector = '.' + this.name_ + '-renderer';
+  return [
+    /* eslint-disable indent */
+    selector + ' .blocklyText {',
+      'cursor: default;',
+      'fill: #fff;',
+      'font-family: sans-serif;',
```

Comment:
This can now use the constants for font-family and size.

Reply:
True!!

Reply:
Fixed.